app/store.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const User = require('./model');
+
+module.exports = ({
+    url
+}) => {
+    return mongoose.connect(url, {
+        useNewUrlParser: true
+    }).then((data) => {
```

Comment:
if you don't need `data` better to remove it from params - clearer and more obvious

Reply:
Will do